src/components/Personalization/index.js

```diff
@@ -15,13 +15,25 @@ import { string, boolean, arrayOf, objectOf } from "../../utils/validation";
 import { initRuleComponentModules, executeRules } from "./turbine";
 import { hideContainers, showContainers } from "./flicker";
 import collectClicks from "./helper/clicks/collectClicks";
+import {
+  DOM_ACTION,
+  HTML_ITEM_CONTENT,
+  JSON_CONTENT_ITEM,
+  REDIRECT_ITEM
+} from "../../constants/schemas";
 
 const DECISIONS_HANDLE = "personalization:decisions";
 const PAGE_WIDE_SCOPE = "page_wide_scope";
 const GET_DECISIONS_OPTIONS_SCHEMA = {
   viewStart: boolean().default(false),
   scopes: arrayOf(string()).default([])
 };
+const ALL_SCHEMAS = [
+  DOM_ACTION,
+  HTML_ITEM_CONTENT,
+  JSON_CONTENT_ITEM,
+  REDIRECT_ITEM
+];
```

Comment:
Rather than having to update this array every time a new schema is added to the constants file, this is what I do:

```
import { values } from "../../utils";
import * as schemasEnum from "../../constants/schemas";

const allSchemas = values(schemasEnum);
```

Reply:
great! thank you